app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -162,6 +167,24 @@ class StatePlayerRecyclerViewAssembler private constructor(
 
   val isCorrectAnswer = ObservableField<Boolean>(false)
 
+  fun saveState(bundle: Bundle) {
+    val hintState: HintState = HintState.newBuilder().apply {
```

Comment:
Interesting way to assemble the builder. Much more Kotlin-esque than what we do elsewhere.

Reply:
Thanks, I like to use scope functions whereever possible